microsoft-365/managed-desktop/get-ready/authentication.md

```diff
@@ -25,22 +25,22 @@ This topic explains the things you need to check in order to ensure that apps an
 
 ## Single Sign-On for on-premises resources
 
-Single Sign-On (SSO) by using UPN and passwords is enabled by default on Microsoft Managed Desktop Devices. But your users can also use Windows Hello for Business, which requires some extra setup steps. 
+Single Sign-On (SSO) by using UPN and password is enabled by default on Microsoft Managed Desktop Devices. But your users can also use Windows Hello for Business, which requires some extra setup steps. 
 
-### Single Sign-On by using UPN and passwords
+### Single Sign-On by using UPN and password
 
 In most organizations, your users will be able to use SSO to authenticate by UPN and password on Microsoft Managed Desktop Devices. However, to make sure this will work, you should double-check the following:
 
 - Confirm that Azure Active Directory (AAD) Connect is set up and uses an on-premises Active Directory server running Windows Server 2008 R2 or later.
 - Confirm that AAD Connect is running a supported version and is set to sync these three attributes with Azure AD: 
     - DNS domain name of the on-premises Active Directory (where the end-users are located)
-    - NetBIOS of yor on-premises Active Directory (where th end-users are located)
+    - NetBIOS of yor on-premises Active Directory (where the end-users are located)
```

Comment:
NetBIOS of **your** on-premises Active Directory (where the end-users are located)

Please do correction as indicated in **bold** (your).